carsReceiver/src/main/java/tkhal.receiver.controller/CarStandardizerController.java

```diff
@@ -0,0 +1,45 @@
+package tkhal.receiver.controller;
+
+import model.Car;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.SQLException;
+import java.time.Duration;
+
+public class CarStandardizerController {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(CarStandardizerController.class);
+    private static Car standardizerCar = null;
+    private static AfterFilterConsumer afterFilterConsumer = new AfterFilterConsumer();
+    private static KafkaConsumer<String, Car> consumer;
+    private static JdbcCarsWriter jdbcCarsWriter = new JdbcCarsWriter();
+    private static CarStandardizer standardizer = new CarStandardizer();
```

Comment:
Do not instantiate classes like this. Use constructor

Reply:
done